Lombiq.Privacy.Tests.UI/Extensions/UITestContextExtensions.cs

```diff
@@ -15,4 +17,16 @@ public static class UITestContextExtensions
 
     public static Task EnablePrivacyRegistrationConsentFeatureAsync(this UITestContext context) =>
         context.EnableFeatureDirectlyAsync(FeatureNames.RegistrationConsent);
+
+    public static Task AcceptPrivacyConsentAsync(this UITestContext context) =>
+        // For Application Insights tracking to be enabled, even in offline mode, this needs to be run.
```

Comment:
```suggestion
        // This is necessary for Application Insights tracking, even in offline mode.
```

Also shouldn't this be in `<remarks>` instead of just a comment?